index.html

```diff
@@ -0,0 +1,2263 @@
+            <i class="fas fa-user-alt"></i>
+            <p id="login-txt">LOGIN</p>
+            <button type="button" class="  subscribe-btn mx-3 ">
+              SUBSCRIBE >
+            </button>
+            <div class="collapse nav-bar-collapse">
+              <ul class="navbar-nav">
+                <li class="nav-item">
+                  <a href="#" class="nav-link">link1</a>
+                  <a href="#" class="nav-link">link1</a>
+                  <a href="#" class="nav-link">link1</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </nav>
+        <!-- second-header -->
+        <div class="d-flex justify-content-start p-1 w-100" id="second-header">
+          <a class="nav-link border-end text-dark"> U.S.</a>
+          <a class="nav-link border-end text-dark">World</a>
+          <a class="nav-link border-end text-dark">Business</a>
+          <a class="nav-link border-end text-dark">Tech & Science</a>
+          <a class="nav-link border-end text-dark">Culture</a>
+          <a class="nav-link border-end text-dark">Newsgeek</a>
+          <a class="nav-link border-end text-dark">Sports</a>
+          <a class="nav-link border-end text-dark">Health</a>
+          <a class="nav-link border-end text-dark">The Debate</a>
+          <a class="nav-link border-end text-dark">Experts</a>
+          <a class="nav-link border-end text-dark">Vantage</a>
+          <a class="nav-link border-end text-dark">
+            Search
+            <i class="fas fa-search"></i>
+          </a>
+        </div>
+        <!-- third-header -->
+        <nav
+          class="navbar bg-light navbar-expand border-bottom w-100 p-1"
+          id="third-header"
+        >
+          <ul class="navbar-nav border-top">
+            <li class="nav-item">
+              <a class="nav-link text-dark"> TRENDING</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link text-secondary">JOE BIDEN</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link text-secondary">DONALD TRUMP</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link text-secondary">CORONAVIRUS</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link text-secondary">STIMULUS PACKAGE</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link text-secondary">VACCINE</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link text-secondary">CAPITOL RIOTS</a>
+            </li>
+          </ul>
+        </nav>
+      </nav>
+      <!-- first section 
+            [FEATURED STORIES], 
+            [TOP STORY, MY TURN, CULTURE & TRAVEL, MORE STORIES] 
+            [THE DEBATE, OPINION]-->
+      <div class="container">
+        <div class="row">
+          <!-- col-1 FEATURED STORIES -->
+          <div class="col-lg-4 col-md-5 col-sm-12">
```

Comment:
- [ ] Use semantic HTML for `<main>`, `<article>`, `<section>`, `<h1>` to `<h6>` for titles, minimize the use of `div` containers they are generic and make it difficult to navigate through the HTML file. Semantic tags improve readability and reduces the complexity of the code. Review your HTML and apply semantic HTML